helm-swoop.el

```diff
@@ -661,7 +672,7 @@ If $linum is number, lines are separated by $linum"
          (ad-enable-advice 'helm-move--previous-line-fn 'around
                            'helm-multi-swoop-previous-line-cycle)
          (ad-activate 'helm-move--previous-line-fn)
-         (add-hook 'helm-update-hook 'helm-swoop--pattern-match)
+         (add-hook 'helm-after-update-hook 'helm-swoop--pattern-match)
```

Comment:
This change does not seem to relate to `New customizations: helm-swoop-flash-region-function`.
Create another PR?

Reply:
It should be done in previous PR https://github.com/emacsorphanage/helm-swoop/pull/145

Maybe I miss something during rebase...